### ISSUE TITLE
help me

### ISSUE THREAD

**Issue by detroldandelion** [Holldean/BERT-Pruning#3]
Hi,
Thank you so much for publishing your code. I download your code and run it step by step (run_all.sh, factorize.sh, run.sh, remove_mask.sh) on the sst-2 task. The last step, 'finetune again', is not provided and I follow the settings in run_all.sh. But the result is unsatisfying (dev acc =83%). And when I use your method on some smaller datasets (e.g., RTE, CoLA), the actual_compact_rate (after removing mask) is larger than 1. (RTE=1.36). Could your please provide more details of sst-2 task about the last step or tell me the most possible reasons, if it is convenient for you.  I would really appreciate it.
Sincerely.

**Comment by Arexh** [comment]
Hi, there. I wrote this project a long time ago, and the original intention was to reproduce the [Wang's](https://arxiv.org/abs/1910.04732) paper. As you can see, the results of my experiment were terrible. At first, the results of my experiment were far inferior to the paper, then I added a fine-tuning step to improve the accuracy (which is not mentioned in this paper). According to my impression, this algorithm is more sensitive to parameters, and the training results fluctuate greatly. I think I can find a time to run my code, but I'm sorry I don't have time to do this recently. 

Thanks for your attention. I will start ASAP!
